src/app/components/add-card/add-card.component.css

```diff
@@ -0,0 +1,13 @@
+.date {
+  position: relative;
+  font-family: 'Roboto';
+  font-style: normal;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  color: #aaacbc;
```

Comment:
We could move color values to CSS variables under `:root` or `:host` (if component specific) for reusability.